src/components/Redoc/Redoc.tsx

```diff
@@ -1,50 +1,113 @@
 import * as PropTypes from 'prop-types';
 import * as React from 'react';
 
-import { ThemeProvider } from '../../styled-components';
-import { OptionsProvider } from '../OptionsProvider';
-
 import { AppStore } from '../../services';
+import { ThemeProvider } from '../../styled-components';
+import { abortRequest } from '../../utils';
```

Comment:
```suggestion
import { AbortRequest } from '../../utils';
```
Have it uppercased like the others

Reply:
`abortRequest` is name of a function. It's OK for it to be defined (and imported) in lowercase.